IncrementSlider/IncrementSlider.module.less

```diff
@@ -9,6 +10,28 @@
 		color: @agate-incrementslider-color;
 		margin: @agate-incrementslider-horizontal-margin;
 		direction: ltr;	// Always arrange this according to LTR rules
+		transform: @agate-slider-bg-transform;
+
+		&::before {
+			content: "";
+			position: absolute;
+			.position(0);
+			background-color: @agate-slider-bg-color;
+			background-image: @agate-slider-bg-image;
+			border-radius: @agate-slider-border-radius;
+		}
+
+		.decrementButton,
+		.incrementButton,
+		.slider {
+			transform: @agate-incrementslider-bg-transform;
+		}
+
+		.slider {
+			&::before {
+				content: none;
+			}
+		}
```

Comment:
Why is this needed? I commented this block locally and I see no difference

Reply:
![image](https://user-images.githubusercontent.com/63341832/117988539-2da70280-b344-11eb-9ccf-a0cc5c917d9d.png)
It might interfere with the other ::before. (see Cobalt night)

Reply:
👍